vital/types/query_result.h

```diff
@@ -90,7 +90,7 @@ class VITAL_EXPORT query_result
 
   void set_temporal_bounds( timestamp const&, timestamp const& );
 
-  void set_location( vital::geo_point );
```

Comment:
This change decreased style consistency and serves no other point; please remove it.